src/main/kotlin/com/criptext/mail/BaseActivity.kt

```diff
@@ -492,7 +491,6 @@ abstract class BaseActivity: PinCompatActivity(), IHostActivity {
 
     override fun exitToScene(params: SceneParams, activityMessage: ActivityMessage?,
                              forceAnimation: Boolean, deletePastIntents: Boolean) {
-        finish()
```

Comment:
Two things:
1. exitToScene is used in almost all the activities. Are you sure that by removing the `finish()` nothing behaves rare?
2. If exitToScene doesn't make you exit the scene, change his name.